pyanaconda/modules/localization/installation.py

```diff
@@ -16,9 +16,21 @@
 # Red Hat, Inc.
 #
 import os
+import shutil
 
-from pyanaconda.modules.common.errors.installation import LanguageInstallationError
+from pyanaconda.modules.common.errors.installation import LanguageInstallationError, \
+    KeyboardInstallationError
 from pyanaconda.modules.common.task import Task
+from pyanaconda.core.constants import DEFAULT_VC_FONT, DEFAULT_KEYBOARD
+from pyanaconda.keyboard import InvalidLayoutVariantSpec
+from pyanaconda.modules.localization.localed import LocaledWrapper
+from pyanaconda.anaconda_loggers import get_module_logger
+
+log = get_module_logger(__name__)
+
+X_CONF_DIR = "/etc/X11/xorg.conf.d"
+X_CONF_FILE_NAME = "00-keyboard.conf"
+VC_CONF_FILE_PATH = "/etc/vconsole.conf"
```

Comment:
These values are defined also in `KeyboardInstallationTask`.

Reply:
Removed form the KeyboardInstallationTask